formats/help/publisher_v2/examples/help_page.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Cookies",
+  "base_path": "/help/cookies",
+  "description": "GOV.UK uses cookies to collect information about how users browse the site",
+  "schema_name": "help",
+  "document_type": "help_page",
+  "need_ids": [],
+  "public_updated_at": "2014-12-16T12:49:50.000+00:00",
+  "publishing_app": "publisher",
+  "rendering_app": "frontend",
+  "routes": [
+    {
+      "path": "/help/cookies",
+      "type": "exact"
+    },
+    {
+      "path": "/help/cookies.json",
+      "type": "exact"
+    }
+  ],
+  "redirects": [],
+  "update_type": "minor",
+  "change_note": null,
+  "details": {
+    "body": [
```

Comment:
It is only sending govspeak. Publishing API seems to helpfully make the HTML on our behalf.